index.mjs

```diff
@@ -8,16 +8,34 @@ async function run() {
     const githubProject = process.env.GITHUB_REPOSITORY
     const githubBranch = githubRef.replace('refs/heads/', '')
     const githubRepo = githubProject.split('/')[1]
-    const teamId = core.getInput('vercel_team_id')
-    const projectId = core.getInput('vercel_project_id')
+    const vApp = core.getInput('vercel_app')
+    const vFrom = core.getInput('vercel_from')
+    const vLimit = core.getInput('vercel_limit')
+    const vProjectId = core.getInput('vercel_project_id')
+    const vSince = core.getInput('vercel_since')
+    const vState = core.getInput('vercel_state')
+    const vTarget = core.getInput('vercel_target')
+    const vTeamId = core.getInput('vercel_team_id')
+    const vTo = core.getInput('vercel_to')
+    const vUntil = core.getInput('vercel_until')
+    const vUsers = core.getInput('vercel_users')
 
-    core.info(`Retrieving deployment preview for ${teamId}/${projectId} ...`)
+    core.info(`Retrieving deployment preview for ${vTeamId}/${vProjectId} ...`)
     const { url, state } = await getDeploymentUrl(
       vercelToken,
       githubRepo,
       githubBranch,
-      teamId,
-      projectId
+      vApp,
```

Comment:
Can we put optional parameters into "options", then the function signature doesn't need to change that often.

Reply:
Correct, that should give more flexibility with optional parameters, what do you think?

Reply:
Makes ense @PatrickHeneise I'll try and get to it this week or my branch should allow you to push new commits too if you wanted to bypass me. I've implemented this in the workflow I needed it in via my fork but I'd like to swap back to yours instead once I get some time to finish this PR.